ZUGFeRD/InvoiceDescriptor22UBLWriter.cs

```diff
@@ -271,84 +272,109 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFo
             }
 
             // PaymentMeans
-            if (this.Descriptor.PaymentMeans != null)
+            if (this.Descriptor.CreditorBankAccounts.Count == 0 && this.Descriptor.DebitorBankAccounts.Count == 0)
```

Comment:
can you please add a test for ubl and multiple bank accounts to make sure that this code works?